lib/task.js

```diff
@@ -80,6 +91,16 @@ class Task {
 		return Promise.resolve()
 			.then(() => {
 				this.state = state.PENDING;
+				return this.skip();
+			})
+			.then(skip => {
+				if (skip) {
+					const reason = typeof skip === 'string' ? skip : `Skipping ${this.title}.`;
```

Comment:
I don't think we should print a reason if no reason was provided by the users themselves.